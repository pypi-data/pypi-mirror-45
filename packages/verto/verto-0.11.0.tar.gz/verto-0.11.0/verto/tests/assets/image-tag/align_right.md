{image file-path="computer-studying-turing-test.png" alt="placeholder image" alignment="right"}
