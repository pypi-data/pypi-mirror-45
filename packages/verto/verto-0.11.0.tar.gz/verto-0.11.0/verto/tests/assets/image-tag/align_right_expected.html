<div>
<img alt="placeholder image" class="right-align" src="{% static 'computer-studying-turing-test.png' %}" />
</div>
